java/kotlin-extractor2/.idea/kotlinc.xml

```diff
@@ -1,13 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="Kotlin2JsCompilerArguments">
+    <option name="moduleKind" value="plain" />
+  </component>
```

Comment:
This is probably not needed, but when any setting is changed in the kotlin compiler section in IDEA, this gets added.